content/blog/2024-05-28-announcing-810-stable.md

```diff
@@ -0,0 +1,100 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/aMvI5E9-LYI?si=8x_HvLo-aSIo5zpM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## ISOs, Live Images, Cloud and Containers
+
+AlmaLinux also offers a variety of Cloud, Container and Live Images. The builds for these get kicked off as soon as the public repository was ready. 
+
+**The following images are expected to be available shortly.** 
+
+* [Container](https://wiki.almalinux.org/containers/) images including Platform and UBIs alternatives. We provide a wide variety of containers for your use. 
+* [LXC/LXD](https://images.linuxcontainers.org/images/almalinux/) 
+* [Live Media](https://wiki.almalinux.org/LiveMedia.html) for GNOME, GNOME-mini, KDE, XFCE, MATE and more.
+* Cloud Images 
+    * [AWS](https://wiki.almalinux.org/cloud/AWS.html) for x86_64 and AArch64 EC2 Instances
+    * [Azure](https://wiki.almalinux.org/cloud/Azure.html) for x86_64 and AArch64 VMs
+    * [Google Cloud](https://wiki.almalinux.org/cloud/Google.html)
+    * [Generic Cloud/Cloud-init](https://wiki.almalinux.org/cloud/Generic-cloud-on-local.html) for all 4 architectures
+    * [OpenNebula](https://wiki.almalinux.org/cloud/OpenNebula.html) for x86_64 and AArch64 architectures
+    * [Oracle Cloud Infrastructure](https://wiki.almalinux.org/cloud/OCI.html) for x86_64 and AArch64 Instances
+* [Vagrant Boxes](https://app.vagrantup.com/almalinux):
+    * Libvirt
+    * VirtualBox
+    * Hyper-V
+    * VMWare for x86_64 and AArch64
+    * Parallels for AArch64
+* [Raspberry Pi](https://wiki.almalinux.org/documentation/raspberry-pi.html)
+* [Windows Subsystem for Linux](https://wiki.almalinux.org/documentation/wsl.html) for x86_64 and AArch64
+
+## Release Notes and More Information
+
+AlmaLinux 8.10 brings updates to security and data protection, and improvements in web-console and system roles to automate operations and ensure consistency in intricate IT settings. The release continues to enhance system availability, reliability, and recovery processes, alongside improving virtual machine snapshot functions in hybrid cloud scenarios. New system roles have been introduced to streamline the creation and administration of logical volume manager (LVM) snapshots for better data backup and recovery processes. Performance, scalability, and reliability continue to be the focus of updates in the 8.10 version to aid developers in application development and management.
+
+You can read the full release notes for this version on the wiki: [AlmaLinux OS 8.10 Release Notes](https://wiki.almalinux.org/release-notes/8.10.html).
```

Comment:
404 for now, https://wiki.almalinux.org/release-notes/8.10-beta.html needs to be renamed before this is published

(PR: https://github.com/AlmaLinux/wiki/pull/419/)

Reply:
Wiki should be updated before merging it
In this case, we will be fine